docs/.vuepress/sidebar/en.js

```diff
@@ -323,6 +321,24 @@ module.exports = [
               "starts_with",
               "strleft",
               "strright",
+              {
```

Comment:
Do we need an extra level?

Reply:
It’s better to put like and not like together